### DISCUSSION TITLE
Only seeing my posts in the Home tab

### DISCUSSION THREAD

**Discussion by vitorpamplona** [KoalaSat/nostros#70] [General]
Is that how it is supposed to be? I can see replies from everyone to my posts, but not the posts from people I am following. Am I missing something?

**Comment by KoalaSat** [comment]
Hi @vitorpamplona are you able to see something on you following section? You can go there by opening your profile tab and tapping the orange circle on the bottom right

**Reply by KoalaSat** [reply]
Also, can you share the version you installed?

**Reply by vitorpamplona** [reply]
I side loaded the last one from the release page. Maybe my user takes too long to load. It took a while to show my followers. The followings are still blank.

**Reply by KoalaSat** [reply]
Is it possible the data from your contacts is not on the relay you are connected to? Try to add 3/4 relays

**Reply by vitorpamplona** [reply]
hum.. so I do have a couple of followers that the app cannot load any additional profile data. Is the app stuck in loading follower profiles? 

Also, why does it load followers before followings? :)

**Reply by vitorpamplona** [reply]
My public key is 460c25e682fda7832b52d1f22d3d22b3176d972f60dcdc3212ed8c92ef85065c if you want to test.

**Reply by KoalaSat** [reply]
Ok thanks I'll check it out with the debugger and let you know

**Reply by KoalaSat** [reply]
I just quickly tried it in my phone and I'm able to see the feed with your contracts, what are the relays you are connected to?

**Reply by vitorpamplona** [reply]
I just added all available gateways in the app. Still stuck. hum...

**Comment by KoalaSat** [comment]
I have been working on a POC to help users during the logging process, I noticed sometimes the relays are under performance and it takes a while  to receive the data, so I decided to create a post-login screen to inform the user about how it's going. 

It on the latest version, maybe this will help you @vitorpamplona

**Reply by vitorpamplona** [reply]
Just downloaded the new version. Here's what I see: https://photos.app.goo.gl/iyMVALKP3DpLvZRJ8

**Reply by vitorpamplona** [reply]
Home is just me. Mentions are updated. PMs are updated. Followings are not loaded. Followers seem to be half loaded.

**Reply by KoalaSat** [reply]
So the problem seems to be that your followers (Pet Names) are not found on the relays. 

Because Nostros is in alpha I only enabled relay management for a closed list of relays, once this is opened you should be able to connect to a relay you actually know contains your data and load it.

In the mid-time, I can only think on using a desktop client to upload this info to one of the relays on the list and obtain it from there.
